python/semantic_kernel/connectors/ai/open_ai/prompt_execution_settings/open_ai_prompt_execution_settings.py

```diff
@@ -82,3 +82,4 @@ class OpenAIEmbeddingPromptExecutionSettings(PromptExecutionSettings):
     extra_query: Optional[Dict] = None
     extra_body: Optional[Dict] = None
     timeout: Optional[float] = None
+    dimensions: Optional[int] = None
```

Comment:
I would like the min and max to be specified using Field, see this: https://docs.pydantic.dev/latest/concepts/fields/#numeric-constraints

Reply:
added numeric constraint. Please check :) Thanks!